README.md

```diff
@@ -139,14 +146,24 @@ Describe to him what a RESTful route is, and list what the seven RESTful routes
 would look like for such an application.
 
 Your description:
-```
+
 Replace this with your answer
-```
+```A RESTful route is an http method with a path for CRUD actions. RESTful methods include new, create, edit, update, destroy, index, show.
```

Comment:
Very close -- you're hitting on the important ideas here. The RESTful methods GET, POST, PUT, PATCH, DELETE -- those methods combined with the appropriate paths are our RESTful routes which are associated with the controller actions Index New Show Edit Create Update and Destroy